cantabular/table.go

```diff
@@ -35,6 +37,37 @@ type Table struct {
 	Rules      Rules       `json:"rules,omitempty"`
 }
 
+type ObservationDimension struct {
+	Dimension   string `bson:"dimension"           json:"dimension"`
+	DimensionID string `bson:"dimension_id"           json:"dimension_id"`
+	Option      string `bson:"option"           json:"option"`
+	OptionID    string `bson:"option_id"           json:"option_id"`
+}
+
+type GetObservationResponse struct {
+	Dimensions  []ObservationDimension `bson:"dimensions"           json:"dimensions"`
+	Observation float32                `bson:"observation"   json:"observation"`
```

Comment:
Indentation could be tidied up here. Additionally - do we need the `bson`? There seems to be a mixture of approaches here.

Reply:
The bson part has now been removed, it had been copied across from previous code.